js/index.js

```diff
@@ -40,3 +41,100 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Navigation Bar Text
+navText = ["Services", "Product", "Vision", "Features", "About", "Contact"];
+const firstChild = document.querySelector("nav a:nth-child(1)");
+  firstChild.textContent = siteContent['nav']['nav-item-1'];
+const secondChild = document.querySelector("nav a:nth-child(2)");
+  secondChild.textContent = siteContent['nav']['nav-item-2'];
+const thirdChild = document.querySelector("nav a:nth-child(3)");
+  thirdChild.textContent = siteContent['nav']['nav-item-3'];
+const forthChild = document.querySelector("nav a:nth-child(4)");
+  forthChild.textContent = siteContent['nav']['nav-item-4'];
+const fifthChild = document.querySelector("nav a:nth-child(5)");
+  fifthChild.textContent = siteContent['nav']['nav-item-5'];
+const sixthChild = document.querySelector("nav a:nth-child(6)");
+  sixthChild.textContent = siteContent['nav']['nav-item-6'];
+
+const navContainer = document.querySelector("nav");
+const seventhChild = document.createElement("a");
+const eighthChild = document.createElement("a");
+seventhChild.textContent = "Investor Relations";
+eighthChild.textContent = "Resources";
+navContainer.appendChild(seventhChild);
+navContainer.prepend(eighthChild);
+navContainer.style.fontSize = ".9rem"
+
+let navBarColor = Array.from(document.querySelectorAll("a"))
+navBarColor.forEach(anchor => anchor.style.color = "green");
```

Comment:
great use of Array.from but remember that using querySelectorAll already returns a Node List / Array.